internal/storage/caching/fast_stat_bucket.go

```diff
@@ -294,3 +295,18 @@ func (b *fastStatBucket) StatObjectFromGcs(ctx context.Context,
 
 	return
 }
+
+func (b *fastStatBucket) GetFolder(
+	ctx context.Context,
+	prefix string) (folder *controlpb.Folder, err error) {
+	// Fetch the listing.
+	folder, err = b.wrapped.GetFolder(ctx, prefix)
+	if err != nil {
+		return
+	}
+
+	// TODO: add folder metadata in stat cache
+	// b.insertFolder(folder)
```

Comment:
It should happen in this PR only ?

Reply:
trying to make small PRs, since not adding this change is not breaking any existing functionality, will be in next PR where Get folder response is stat cached